status/boost_check_library.py

```diff
@@ -83,6 +107,33 @@ def check_organization_include(self):
                 ''',
                 'org-inc-one')
     
+    def check_organization_macros(self):
+        libdir = os.path.join(self.library_dir,'include','boost',self.library_name)
+        if os.path.isdir(libdir):
+            badmacros = {}
+            for root, dirs, files in os.walk(libdir):
+                if os.path.sep + 'detail' + os.path.sep in root or root.endswith(os.path.sep + 'detail'):
+                    continue
+                for item in files:
+                    ext = os.path.splitext(item)
+                    if ext[1] == '.hpp' or ext[1] == '.ipp' or ext[1] == '.h':
+                        p = Preprocessor(self)
+                        path = os.path.join(root, item)
+                        if self.debug:
+                            print(">>> preprocessing:", path)
+                        with open(path, 'rt') as ih:
+                            p.parse(ih)
+                        while p.token():
+                            pass
+                        for macro in p.macros:
+                            if not macro in badmacros and not macro.startswith('BOOST_') and not macro.startswith('__'):
+                                badmacros[macro] = path
+                                if p.macros[macro].source:
+                                    badmacros[macro] = "%s:%d" % (p.macros[macro].source, p.macros[macro].lineno)
+            if badmacros:
```

Comment:
Not required. looping over an empty dict is fine an a no-op

Reply:
Fixed